qmt/__init__.py

```diff
@@ -1,4 +1,9 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+
+from ._version import __version__
+
+del _version
```

Comment:
Sorry what does this del do? Does `from ._version import __version__` create a `_version` variable?

Reply:
Exactly what you say. It's just to not confuse the users.